packages/dev/docs/pages/releases/2023-09-07.mdx

```diff
@@ -0,0 +1,291 @@
+
+In addition to the above API changes, we also released the following improvements:
+
+
+- New [Styling](https://react-spectrum.adobe.com/react-aria/styling.html) page in the docs that gives an overview of the various methods to style React Aria Components, including states, render props, slots, and animation with CSS keyframes and Framer Motion.
+- A new [Tailwind CSS plugin](https://react-spectrum.adobe.com/react-aria/styling.html#plugin) that gives you shorter modifier names and autocomplete for the states of React Aria Components.
+- Our collection components now render on the server during SSR.
+- Updates to the documentation of every component:
+    - The anatomy section has been expanded to include improved visuals and the corresponding JSX structure for every element and supported slot within a component.
+    - The content has been reorganized into sub sections and ordered to place reference material at the end of the page, and move more examples toward the top.
+    - The “Advanced Customization” section has been updated to the public contexts provided and consumed by each component, and how you can use these to create your own higher level patterns.
+    - Documentation has been grouped into sections in the left navigation for Components, Hooks, Interactions, and Utilities to make it easier to find what you’re looking for.
+
+Thanks to everyone who tried out React Aria Components while it was in alpha! Your feedback has been extremely valuable as we iterate toward a full release.
+
+## Validation state deprecation
+
+In React Spectrum and React Aria hooks, we have deprecated the `validationState` prop in many components, and replaced it with the `isInvalid` boolean prop. Most components do not include a visual  “valid” state, so the previous API was confusing. For React Spectrum, field-like components such as `TextField`  and `ComboBox` continue to support the `validationState` prop to display a green checkmark, but other components such as `CheckboxGroup` now use `isInvalid` instead. In React Aria, all hooks have migrated to `isInvalid`. The old API will continue to work in both but is marked as deprecated, and you can update in your own time.
+
+## Fixes
+- Add checks to prevent event bubbling in `useDatePickerGroup` - [@yihuiliao](https://github.com/yihuiliao) - [PR](https://github.com/adobe/react-spectrum/pull/4956)
+- Do not close overlays on blur if relatedTarget is null - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4931)
+- Removing `aria-disabled` so TalkBack users can open the context dialog - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/4710)
+- Remove erroneous accessibility color contrast ignore rule - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/5029)
```

Comment:
We can remove this one, this was specific to our own Storybook tests and wasn't present in the last release